.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,13 +10,44 @@
   - when this PR is merged.
   -->
 
```

Comment:
Similarly, let's move the checklist up to here, and call it "Essential Checklist", similar to https://github.com/oppia/oppia/edit/develop/.github/PULL_REQUEST_TEMPLATE.md

Reply:
Done.